docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-role.md

```diff
@@ -0,0 +1,93 @@
+---
+title: role resource
+headerTitle: ybm role
+linkTitle: role
+description: YugabyteDB Managed CLI reference Role resource.
+headcontent: Manage roles
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-cli-role
+    parent: managed-cli-reference
+    weight: 20
+type: docs
+---
+
+Use the `role` resource to create and manage YugabyteDB Managed account [roles](../../../../cloud-admin/managed-roles/).
+
+## Syntax
+
+```text
+Usage: ybm role [command] [flags]
+```
+
+## Examples
+
+List roles in YugabyteDB Managed:
+
+```sh
+ybm role list
+```
+
+Describe the Admin role:
+
+```sh
+ybm role describe --role-name admin
+```
+
+You can use this command to view all the available permissions.
+
+Create a role:
+
+```sh
+ybm role create --role-name backuprole \
+  --permissions resource-type=BACKUP,operation-group=CREATE \
+  --permissions resource-type=BACKUP,operation-group=DELETE \
+```
+
+## Commands
+
+### create
+
+Create a custom role.
+
+| Flag | Description |
+| :--- | :--- |
+| --role-name | Required. Name for the role. |
+| --permissions | Required. Permissions for the role, provided as key value pairs. Permissions are made up of a resource type and an operation group.<br>Arguments:<ul><li>resource-type - the resource to which the permission applies</li><li>operation-group - the operation that the permission allows on the resource</li></ul>Both resource-type and operation-group are mandatory. Specify multiple permissions by using multiple --permissions arguments. Use `ybm role describe --role-name admin` to view a list of all permissions. |
```

Comment:
> Use `ybm role describe --role-name admin` to view a list of all permissions

Do we mean "Use `ybm permission list` to view a list of all available permissions"?

Reply:
Ya. That was added after I first wrote this and this was my workaround